src/site/stages/build/index.js

```diff
@@ -214,6 +232,15 @@ function build(BUILD_OPTIONS) {
     'Apply layouts',
   );
 
+  smith.use(
+    emitToFiles(
+      BUILD_OPTIONS,
+      path.resolve(__dirname, '../../../../build/', 'after-layouts'),
+      file => file.contents,
+    ),
+    'Emit contents of files to build/after-layouts',
+  );
```

Comment:
I think this could get lost in the noise of the dozens of other build steps, and someone new might get confused trying to understand how these files get used or why we care about them.

Reply:
Yeah, that's true. If I merge this into `master`, I'll likely remove all uses of the plugin from the build script in favor of a single commented example since this is only to aid in debugging. I'll also add proper documentation on the function. :+1: 

For now, this was intended to help @drorva understand how the process works by seeing the output at various stages in the Metalsmith pipeline.